fuzzing/scripts/custom-build.sh

```diff
@@ -68,9 +68,13 @@ fi
 # rebuild shortcut:
 
 if [[ "${opt_rebuild}" == "1" ]]; then
-    bash build/glog.sh       --no-init
+    bash build/zlib.sh       --no-init
```

Comment:
Maybe sort this alphabetically if possible?  The same for other such longer lists elsewhere.

Reply:
The order here is more about dependency order (maybe there should be a comment about that). We want to build these from fewest dependencies (lowest level) to most dependencies (highest level). In theory we could actually list out the dependencies and build some of these in parallel. This is unfortunately basically building a sanitized distribution. It would be nice if oss-fuzz had the manpower to be rid of the silly Ubuntu docker image and instead build some distro like Arch with the entire userspace built with particular sanitizers. Which is what is being cobbled together here. For example, if we wanted to put glog back into this list is would need to also go after libcxx since it would also need to be built with the sanitizer built libc++.